packages/@lwc/compiler/src/__tests__/fixtures/expected-minify-no-comments.js

```diff
@@ -1 +1 @@
-define("x/comments",["lwc"],function(e){var t=[function(e,t,s){return(s?":host{color:blue}":e+"{color:blue}")+"x-comments"+t+"{color:green}"}];function s(e,t,s,n){const{d:o,h:r}=e;return[r("h1",{key:0},[o(t.myname)])]}var n=e.registerTemplate(s);s.stylesheets=[],t&&s.stylesheets.push.apply(s.stylesheets,t),s.stylesheetTokens={hostAttribute:"x-comments_comments-host",shadowAttribute:"x-comments_comments"};class o extends e.LightningElement{constructor(...e){super(...e),this.myname="default value"}}return e.registerDecorators(o,{track:{myname:1}}),e.registerComponent(o,{tmpl:n})});
```

Comment:
any reason for this test to be removed?

Reply:
I suspect it is a behavior change that might not be observable to the end-user since it is about comments.

Reply:
If you look into the diff, it's a line replacement, not a removal. The arguments identifier have been swapped out.